pkg/referrerstore/oras/utils.go

```diff
@@ -53,3 +53,13 @@ func OciDescriptorToReferenceDescriptor(ociDescriptor oci.Descriptor) ocispecs.R
 		ArtifactType: ociDescriptor.ArtifactType,
 	}
 }
+
+func OciManifestToReferenceManifest(ociManifest oci.Manifest) ocispecs.ReferenceManifest {
+	return ocispecs.ReferenceManifest{
```

Comment:
Seems the `Layers` field of `ReferenceManifest` is not used now, wonder if we need to remove it.

Reply:
I believe I have already removed it from the struct. Is there somewhere else I'm missing?

Reply:
oops, didn't notice it.